src/matlab2tikz.m

```diff
@@ -435,11 +435,11 @@ function matlab2tikz(varargin)
 
     % Alternative Positioning of axes.
     % Select relevant Axes and draw them.
-    [relevantAxesHandles, axesBoundingBox] = getRelevantAxes(m2t, axesHandles);
+    [m2t, axesBoundingBox] = getRelevantAxes(m2t, axesHandles);
```

Comment:
I see now that you needed it before your most recent commit cf4f961d39bd24ec3c9402fd502e0804c2e535d0.